cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,74 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="function")
+def config_manager_rest(config_manager_obj):
+    """Creates provider using REST API."""
+    config_manager_obj.create_rest()
+    assert_response(config_manager_obj.appliance)
+    rest_entity = config_manager_obj.rest_api_entity
+    rest_entity.reload()
+
+    yield rest_entity
+
+    if rest_entity.exists:
+        rest_entity.action.delete()
+        rest_entity.wait_not_exists()
+
+
+@pytest.mark.tier(3)
+def test_config_manager_create(config_manager_rest):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: medium
+        initialEstimate: 1/10h
+        testSteps:
+            1. Add Ansible Tower via REST
+        expectedResults:
+            1. Provider must be added and validated successfully.
+
+    Bugzilla:
+        1621888
```

Comment:
This BZ reference could be removed now that the BZ is closed.

Reply:
I have kept it as metadata, just for a reference.